canonical-web-and-design/etclite.io
# Replace Jekyll with 11ty and add dotrun check

## Done

Replaced Jekyll with 11ty and removed Ruby dependencies

## QA

- Checkout this branch
- `dotrun clean && dotrun`
- Browse website at http://localhost:8028
- Check all links are working
- Make sure Github checks pass
- Make sure global navigation on top appears and there are no errors in the console


Fix #19
